Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/task.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
```

Comment:
we wanted to update Minor version.
Some customers modified their task locally as a work around for SQL 2016. They might have updated patch version. let's update Minor version